.pre-commit-config.yaml

```diff
@@ -7,7 +7,7 @@ repos:
     hooks:
     - id: black
 -   repo: https://github.com/asottile/pyupgrade
-    rev: v2.29.1
+    rev: v2.31.1
```

Comment:
I'm not sure if this should get squashed into this PR or might be better to have in its own or as part of a general maintenance PR. Not a big deal, just thinking out loud!